src/ray/gcs/test/subscription_executor_test.cc

```diff
@@ -1,225 +0,0 @@
-// Copyright 2017 The Ray Authors.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//  http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-#include "ray/gcs/subscription_executor.h"
-
-#include "gtest/gtest.h"
-#include "ray/gcs/callback.h"
-#include "ray/gcs/entry_change_notification.h"
-#include "ray/gcs/redis_gcs_client.h"
-#include "ray/gcs/test/accessor_test_base.h"
-
-namespace ray {
-
-namespace gcs {
-
-class SubscriptionExecutorTest : public AccessorTestBase<ActorID, ActorTableData> {
- public:
-  typedef SubscriptionExecutor<ActorID, ActorTableData, LogBasedActorTable>
-      ActorSubExecutor;
-
-  virtual void SetUp() {
-    AccessorTestBase<ActorID, ActorTableData>::SetUp();
-
-    actor_sub_executor_.reset(new ActorSubExecutor(gcs_client_->log_based_actor_table()));
-
-    subscribe_ = [this](const ActorID &id, const ActorTableData &data) {
-      const auto it = id_to_data_.find(id);
-      ASSERT_TRUE(it != id_to_data_.end());
-      --sub_pending_count_;
-    };
-
-    sub_done_ = [this](Status status) {
-      ASSERT_TRUE(status.ok()) << status;
-      --do_sub_pending_count_;
-    };
-
-    unsub_done_ = [this](Status status) {
-      ASSERT_TRUE(status.ok()) << status;
-      --do_unsub_pending_count_;
-    };
-  }
-
-  virtual void TearDown() {
-    AccessorTestBase<ActorID, ActorTableData>::TearDown();
-    ASSERT_EQ(sub_pending_count_, 0);
-    ASSERT_EQ(do_sub_pending_count_, 0);
-    ASSERT_EQ(do_unsub_pending_count_, 0);
-  }
-
- protected:
-  virtual void GenTestData() {
-    for (size_t i = 0; i < 100; ++i) {
-      std::shared_ptr<ActorTableData> actor = std::make_shared<ActorTableData>();
-      actor->set_max_restarts(1);
-      actor->set_num_restarts(0);
-      JobID job_id = JobID::FromInt(i);
-      actor->set_job_id(job_id.Binary());
-      actor->set_state(ActorTableData::ALIVE);
-      ActorID actor_id = ActorID::Of(job_id, RandomTaskId(), /*parent_task_counter=*/i);
-      actor->set_actor_id(actor_id.Binary());
-      id_to_data_[actor_id] = actor;
-    }
-  }
-
-  size_t AsyncRegisterActorToGcs() {
```

Comment:
We deleted `AsyncRegister`, and this test case needs to use x, so we deleted `AsyncRegister`. BTW: SubscriptionExecutor is not used, we will delete it later.